js/web/lib/wasm/binding/ort-wasm.d.ts

```diff
@@ -43,6 +43,8 @@ export interface OrtWasmModule extends EmscriptenModule {
   _OrtCreateRunOptions(logSeverityLevel: number, logVerbosityLevel: number, terminate: boolean, tag: number): number;
   _OrtAddRunConfigEntry(runOptionsHandle: number, configKey: number, configValue: number): number;
   _OrtReleaseRunOptions(runOptionsHandle: number): void;
+
+  _OrtEndProfiling(sessionHandle: number): number;
```

Comment:
Do we need `_OrtStartProfiling` as well?

Reply:
ORT C API doesn't support StartProfiling yet. Profiling is started when a session is created.